pkg/rate/api_limiter.go

```diff
@@ -435,14 +435,13 @@ func (l *APILimiter) delayedAdjustment(current, min, max float64) (n float64) {
 }
 
 func (l *APILimiter) calculateAdjustmentFactor() float64 {
-	f := l.params.EstimatedProcessingDuration.Seconds() / l.meanProcessingDuration
-	if f > l.params.MaxAdjustmentFactor {
-		f = l.params.MaxAdjustmentFactor
-	}
-	if f < 1.0/l.params.MaxAdjustmentFactor {
-		f = 1.0 / l.params.MaxAdjustmentFactor
-	}
-	return f
+	return max(
```

Comment:
probably correct but I feel like the new code is less readable?

Reply:
Rewritten to

```go
f := l.params.EstimatedProcessingDuration.Seconds() / l.meanProcessingDuration
f = min(f, l.params.MaxAdjustmentFactor)
f = max(f, 1.0/l.params.MaxAdjustmentFactor)
return f
```

which IMO somewhat retains readability while still switching to use `min`/`max`.